### PR TITLE
FEAT: publish all artifacts during GitHub release

### PR DESCRIPTION
**Pull request by jorgepiloto**
Solve #57 by publishing during a GitHub release all generated artifacts.

### CODE REVIEW

File: release-github/action.yml [hunk 1]
```diff
@@ -46,4 +70,5 @@ runs:
           ${{ inputs.library-name }}/*.whl
           ${{ inputs.library-name }}/*.tar.gz
           documentation-html.zip
-          documentation-pdf/*.pdf
+          documentation-pdf.zip
+          other-artifacts.zip
```
[the remark this answers is not in the document]
Comment (jorgepiloto) [reply release-github/action.yml:74]: Shouldn't it have only one `*` in front of the wheelhouse?

```suggestion
          documentation-pdf.zip
          other-artifacts/*-wheelhouse-*.zip
```

[the remark this answers is not in the document]
Comment (jorgepiloto) [reply release-github/action.yml:74]: Well, in fact, it should be the following:

```suggestion
          documentation-pdf.zip
          other-artifacts/**/*-wheelhouse-*.zip
```

File: release-github/action.yml [hunk 2]
```diff
@@ -35,6 +35,30 @@ runs:
...
+        rm -rf other-artifacts/documentation-html
+        rm -rf other-artifacts/documentation-pdf
+        rm -rf other-artifacts/${{ inputs.library-name }}-artifacts
+
+    - name: "Zip other artifacts"
+      uses: vimtor/action-zip@v1
+      with:
+        files: other-artifacts
+        dest: other-artifacts.zip
+
```
[the remark this answers is not in the document]
Comment (jorgepiloto) [reply release-github/action.yml]: Applying this suggestion. Also, I removed the zip compression step for the other artifacts.

File: release-github/action.yml [hunk 3]
```diff
@@ -46,4 +64,5 @@ runs:
           ${{ inputs.library-name }}/*.whl
           ${{ inputs.library-name }}/*.tar.gz
           documentation-html.zip
-          documentation-pdf/*.pdf
+          documentation-pdf.zip
+          other-artifacts/**/*-wheelhouse-*.whl
```
Comment (RobPasMue) [inline release-github/action.yml:68]: I think this should still be ZIP... not whl
```suggestion
          other-artifacts/**/*-wheelhouse-*.zip
```

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: release-github/action.yml (hunk 3 above)
Comment (jorgepiloto) [reply release-github/action.yml:68]: I thought artifacts were decompressed when downloaded inside of the actions.

Comment (jorgepiloto) [reply release-github/action.yml:68]: Ops, sorry. It looks like the wheelhouse already ships as a ZIP. Makes sense, it is a big collection of wheels...

**Review by RobPasMue — APPROVED** [review]
LGTM @jorgepiloto!! Let's go for its release :)

**Pull request merged**